app/html.cjsx

```diff
@@ -21,7 +21,7 @@ class Application extends React.Component
         <script dangerouslySetInnerHTML={ __html: @props.polyfill }/>
         <script dangerouslySetInnerHTML={ __html: @props.state }/>
         <script dangerouslySetInnerHTML={ __html: @props.config }/>
-        <script dangerouslySetInnerHTML={ __html: @props.locale }/>
+        <script dangerouslySetInnerHTML={ __html: @props.windowLocale }/>
```

Comment:
What about changeing this to `'window.locale="' + @props.locale + '"'` to save a prop.